op-defender/psp_executor/README.md

```diff
@@ -45,27 +92,29 @@ To monitor the _PSPexecutor service_ the metrics server can be enabled. The metr
 The metrics are using **Prometheus** and can be set with the following options:
 | Option | Description | Default Value | Environment Variable |
 | ------------------- | ------------------------- | ------------- | ----------------------------- |
-| `--metrics.enabled` | Enable the metrics server | `false` | `$MONITORISM_METRICS_ENABLED` |
-| `--metrics.addr` | Metrics listening address | `"0.0.0.0"` | `$MONITORISM_METRICS_ADDR` |
-| `--metrics.port` | Metrics listening port | `7300` | `$MONITORISM_METRICS_PORT` |
+| `--metrics.enabled` | Enable the metrics server | `false` | `DEFENDER_METRICS_ENABLED` |
+| `--metrics.addr` | Metrics listening address | `"0.0.0.0"` | `$DEFENDER_METRICS_ADDR` |
+| `--metrics.port` | Metrics listening port | `7300` | `$DEFENDER_METRICS_PORT` |
 
 ### 4. Options and Configuration
 
 The current options are the following:
 
 ```
 OPTIONS:
-   --rpc.url value             Node URL of a peer (default: "http://127.0.0.1:8545") [$PSPEXECUTOR_MON_NODE_URL]
-   --privatekey value          Private key of the account that will issue the pause () [$PSPEXECUTOR_MON_PRIVATE_KEY]
-   --receiver.address value    The receiver address of the pause request. [$PSPEXECUTOR_MON_RECEIVER_ADDRESS]
-   --port.api value            Port of the API server you want to listen on (e.g. 8080). (default: "8080") [$PSPEXECUTOR_MON_PORT_API]
-   --data value                calldata to execute the pause on mainnet with the signatures. [$PSPEXECUTOR_MON_CALLDATA]
-   --log.level value           The lowest log level that will be output (default: INFO) [$MONITORISM_LOG_LEVEL]
-   --log.format value          Format the log output. Supported formats: 'text', 'terminal', 'logfmt', 'json', 'json-pretty', (default: text) [$MONITORISM_LOG_FORMAT]
-   --log.color                 Color the log output if in terminal mode (default: false) [$MONITORISM_LOG_COLOR]
-   --metrics.enabled           Enable the metrics server (default: false) [$MONITORISM_METRICS_ENABLED]
-   --metrics.addr value        Metrics listening address (default: "0.0.0.0") [$MONITORISM_METRICS_ADDR]
-   --metrics.port value        Metrics listening port (default: 7300) [$MONITORISM_METRICS_PORT]
-   --loop.interval.msec value  Loop interval of the monitor in milliseconds (default: 60000) [$MONITORISM_LOOP_INTERVAL_MSEC]
-   --help, -h                  show help
+   --rpc.url value                   Node URL of a peer (default: "http://127.0.0.1:8545") [$PSPEXECUTOR_NODE_URL]
+   --privatekey value                Privatekey of the account that will issue the pause transaction [$PSPEXECUTOR_PRIVATE_KEY]
```

Comment:
Maybe would be better to have the option to have the private key also in a file as well?

Reply:
Interesting, for the current specs the idea was to fetch the secret from outside.
but agreed for standardization in future PR during improvements.